lab-michael/model/client.js

```diff
@@ -0,0 +1,13 @@
+//Client Constructor
+
+'use strict';
+
+const uuid = require('node-uuid');
+
+const Client = module.exports = function(socket) {
```

Comment:
Best to assign this function directly to `module.exports` since it's the only function in this module. Don't be afraid to improve on the starter code!